protected/humhub/modules/user/models/forms/AccountDelete.php

```diff
@@ -68,12 +68,13 @@ public function performDelete()
             return false;
         }
 
-        $this->user->status = User::STATUS_DISABLED;
+        $this->user->status = User::STATUS_SOFT_DELETED;
         $this->user->save();
 
+        Auth::deleteAll(['user_id' => $this->user->id]);
```

Comment:
Do we really need this? If it is deleted anyway, during the `softDelete()` in ActiveJob?